### ISSUE TITLE
Color blindness accessibility of document figures & tables

### ISSUE THREAD

**Issue by sadielbartholomew** [cf-convention/cf-conventions#404]
The colours used to convey, or help with the absorption of, information for some figures and tables in the main Conventions document are not sufficiently accessible to those with some facet of colour blindness, as evidenced below, and in general all should be checked for accessibility in this respect and improved if not up to scratch.

I claim lack of accessibility because, whilst thinking about the presentation of the document in line with the theme of a recent hackathos at the CF Workshop 2022, I thought to run relevant parts (namely, those with figures and tables utilising colour) through some basic checks made possible from free online tools, notably [this very useful 'Colorblind Web Page Filter' tool](https://www.toptal.com/designers/colorfilter). From this basic check I noted that most of the figures were good enough to be comprehended properly in any case of colour blindness, but there were some that in some cases used colours to highlight key information where certain colours weren't distinguishable, as covered below.

I would add that I see from the Issue Trackers and discussion that colour blindness has been considered for the creation of diagrams in certain cases, e.g. https://github.com/cf-convention/cf-conventions/issues/193#issuecomment-525761929, which is great! Can we, as a separate but related issue, add something to the contributing guidelines perhaps to ensure that it is a criteria for any new diagrams that they pass as god enough (to some standard) with respect to this sort of accessibility? MAybe we can use the tool above, or a similar one, as a means to validate this?

**Comment by erget** [comment]
@sadielbartholomew I think this is a really nice initiative. I myself am colourblind and mostly resign myself to not being able to interpret aspects of diagrams out of laziness - I don't like always making noise about it. It's not a huge barrier, but it often makes me a little sad when I can't grok diagrams as fast/well as other people can. What I mean is this is appreciated ideologically and personally :)

So if I understand your proposal it's 2-fold:
1. Update figures to improve accessibility
2. Update `CONTRIBUTING.md` to ask people to check figures if they use colours, referring to some common tools that may be useful

Is that right?

**Comment by sadielbartholomew** [comment]
We discussed this Issue at the [CF Workshop 25 Hackathon session '(Digital) accessibility of the CF Convention document](https://cfconventions.org/Meetings/2025-Workshop.html)' and have made progress on it, so we am hoping we will be able to start putting in PRs to the branch https://github.com/cf-convention/cf-conventions/tree/accessibility-main (along with other digital accessibility improvements from #606), aiming to eventually have the branch ready to raise as a PR against the `main` branch for full incorporation (closing these two issues) when merged.

**Comment by larsbarring** [comment]
Candidate for the CF2026 Housekeeping session ?
